### DISCUSSION TITLE
Can't see the Themes section in my Django Admin

### DISCUSSION THREAD

**Discussion by javidfm** [fabiocaccamo/django-admin-interface#336] [Q&A]
I have all the permissions (even tried being Superuser) and I can't make the Themes section appear in the Django Admin so I can edit the theme..

Any tips?
Thanks!

**Comment by fabiocaccamo** [comment]
@javidfm I suggest you to ensure that you have followed the installation instructions correctly:
https://github.com/fabiocaccamo/django-admin-interface#installation
